fmn/rules/taskotron.py

```diff
@@ -29,8 +38,14 @@ def taskotron_task(config, message, task=None):
     if not task:
         return False
 
-    tasks = [item.strip().lower() for item in task.split(',')]
-    return message['msg']['task'].get('name').lower() in tasks
+    name = message['msg']['task'].get('name')
```

Comment:
Because testcase names are not case-insensitive, afaik. I'd say it's better to change now than later. But if you think it's a minor issue which is likely not going to occur in the future, no problem, I'll put it back.